### PR TITLE
os reference code document

### PR DESCRIPTION
**Pull request by xunpan**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Specify your PR reviewers**:

/cc @gyliu513 @jichenjc @xunpan

**Special notes for your reviewer**:

/sig ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,17 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
    ***NOTE*** The private key used to connect to the master node is by default (currently hardcoded) at ~/.ssh/id_ibmcloud, so make sure you copy the generated key's private key info into this file.
 
+   #### Quick notes on machines.yaml
```
Comment (gyliu513) [inline README.md]: Please run `doctoc`

Comment (jichenjc) [reply README.md]: I prefer to migrate those info into a tips or cloudinfo helper file in doc subfolder...

**Comment by xunpan** [comment]
I add a new document for IBM cloud related setting material or Q&As. `README.md` is a good start for step by step deployment instructions. This can be a good start to organize more detailed IBM cloud related information.
`doctoc` is runned for both `README` and `ibmcloud_help` document.

@gyliu513 @jichenjc

**Pull request merged**